scripts/screeps_rl_experiment_card.py

```
@@ -203,6 +537,11 @@ def validate_card(raw: Any) -> None:
     validate_reward_model(raw.get("reward_model"))
     validate_strategy_variants(first_present(raw, ("strategy_variants", "strategyVariants", "variants")))
     validate_simulation(first_present(raw, ("simulation", "simulator")))
+    policy_gradient = first_present(raw, ("policy_gradient", "policyGradient"))
+    if training_approach == "policy_gradient" and policy_gradient is None:
+        raise CardValidationError("policy_gradient metadata is required when training_approach is policy_gradient")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)